app/views/admin/categories/_category.html.erb

```diff
@@ -1,6 +1,6 @@
     <tr>
```

Comment:
You can format your table like this:
![image](https://user-images.githubusercontent.com/20507565/45129335-c2312e00-b1b5-11e8-84a9-7592e657ba89.png)